MVP_rev1.py

```diff
@@ -0,0 +1,1168 @@
+'''''
+Interactive Programming: Visualizing Music
+
+author@ Maggie Rosner & Aurora Bunten
+
+03/01/17
+'''''
+
+import pygame
+import pyaudio
+import wave
+import sys
+
+class Shape(object):
+    '''
+    the shaps class creates objects that are defined by the note they represent
+    '''
+    def __init__(self, note):
+        self.note = note
+        self.disp = False
+
+
+class Controller(object):
+    '''
+    The controller class calls a specific shape object based on the key pressed.
+    the shape is associated with a specific note which will change the shape_disp
+    to True and play the wave file associated with the note.
+    '''
+    def __init__(self, model):
+        self.model = model
+        self.octave = 4     #sets the default octave to the 4th (middle C)
+    def input_event(self,event):
+        is_audio_event = False  #sets the initial condition for an event (shape display) to False
+        '''
+        the conditions below set the octave based on certain number keys that are pressed
+        '''
+        if event.type == pygame.KEYDOWN: #if a key is pressed
```

Comment:
It's good that you are commenting these lines but I think the level of details here is little too much. I think something like `#determining octave level through user key press` is sufficient enough. The level of readability vs. clarity is always a complicated issue :confused: